webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx

```diff
@@ -234,7 +234,9 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
           )}
         </>
       ) : null}
-      {contextHolders}
+      {contextHolders.map((contextHolder, i) => (
+        <React.Fragment key={i}>{contextHolder}</React.Fragment>
+      ))}
```

Comment:
I think this works, but doesn't each `contextHolder` themselves already have a `<>` wrapper around them? I don't have anything better to suggest, so if this works to get rid of the missing key warning 👍

Reply:
the suggestion didnt work 🥲